### ISSUE TITLE
Insync don't open again after config and close

### ISSUE THREAD

**Issue by ajmasia** [NixOS/nixpkgs#244813]
### Describe the bug
I have installed the latest unstable version of Insync. I start it, configure my account, and the following happens:
- No systry icon appears
- When I close the Insync window and try to start it again, it does nothing
- If I execute any command from the console, I get this message: _Insync doesn't seem to be running. Start it first_.

IMPORTANT: It's seems sync continue working but I can't show the Insync window.

### Steps To Reproduce
1. Install package
2. Start package from Rofi
3. Config account and all works
4. Close Insync window
5. Impossible show the Insync window again to view files or change config

### Expected behavior
I should be able to reopen the Insync window and also use the commands from the console.

### Logs
```bash
 ~ ✦ ➜ insync start --no-daemon --debug 0
QSystemTrayIcon::setVisible: No Icon set
INFO     2023-07-22 13:10:39,126 [mainlogs:_log_run:138] Core(app_version=3.8.6.50504, platform=Linux-x86_64-nixos/23.05) initialized
WARNING  2023-07-22 13:10:41,143 [base_events:_run_once:1782] Executing <Task finished coro=<SettingsMain._load_settings() done, defined at ideskcore/mainsettings.py:186> exception=DBLockedError('The database file is currently locked (SettingsDBInstaller).') created at ideskcore/mainsettings.py:160> took 2.009 seconds
INFO     2023-07-22 13:10:41,145 [core:init:31] DB is locked!
ERROR    2023-07-22 13:10:41,145 [platui_impl:display_notification:82] Cannot initialize notifications.
ERROR    2023-07-22 13:10:44,146 [base_events:default_exception_handler:1619] Task exception was never retrieved after 3.0 seconds
future: <Task finished coro=<SyncMain._start_syncing() done, defined at ideskcore/mainsync.py:199> exception=DBLockedError('The database file is currently locked (SettingsDBInstaller).') created at ideskcore/mainsync.py:185>
source_traceback: Object created at (most recent call last):
  File "threading.py", line 890, in _bootstrap
  File "threading.py", line 926, in _bootstrap_inner
  File "threading.py", line 870, in run
  File "ideskasync/coreloop.py", line 273, in run_loop
  File "asyncio/base_events.py", line 541, in run_forever
  File "asyncio/base_events.py", line 1778, in _run_once
  File "asyncio/events.py", line 88, in _run
  File "ideskcore/core.py", line 27, in init
  File "ideskcore/insync_core.py", line 17, in __init__
  File "ideskcore/mainsync.py", line 185, in __init__
Traceback (most recent call last):
  File "ideskdb/clientdb.py", line 149, in __init__
  File "ideskdb/clientdb.py", line 249, in execute
sqlite3.OperationalError: database is locked

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "ideskcore/core.py", line 28, in init
  File "ideskcore/mainbase.py", line 14, in init
  File "ideskcore/insync_core.py", line 58, in _init
  File "ideskcore/mainsync.py", line 196, in _init
  File "ideskcore/mainhttp.py", line 110, in _init
  File "ideskcore/mainsettings.py", line 164, in _init
  File "ideskcore/mainsync.py", line 200, in _start_syncing
  File "ideskcore/mainsettings.py", line 204, in _load_settings
  File "ideskcore/mainsettings.py", line 25, in __init__
  File "ideskdb/clientdb.py", line 197, in __init__
ideskdb.clientdb.DBLockedError: The database file is currently locked (SettingsDBInstaller).
INFO     2023-07-22 13:10:45,148 [app:quit:80] quitting
INFO     2023-07-22 13:10:45,150 [app:stop:75] core stopped
INFO     2023-07-22 13:10:45,150 [app:stop:77] clean exit
```

### Notify maintainers
@hellwolf 

### Metadata
- system: `"x86_64-linux"`
- host os: `Linux 6.1.34, NixOS, 23.05 (Stoat), 23.05.20230616.c7ff1b9`
- multi-user?: `yes`
- sandbox: `yes`
- version: `nix-env (Nix) 2.13.3`
- nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

**Comment by ajmasia** [comment]
I discover more things about this:
- It's seems the service is running! 
- The systry icon is invisble and stay there. When I click on the invisble char the Insync open
- Any terminal command over Insync works

![image](https://github.com/NixOS/nixpkgs/assets/23419327/f8c68539-b81d-4f1a-ae18-f2f31c7e1fbc)

**Comment by hellwolf** [comment]
Thanks for the report @ajmasia , I will be taking a look.

**Comment by OliverSchr** [comment]
I have the same.
and i tried with x11 and wayland, with plasma and gnome, and with configuration and with just closing the welcome screen.. Always the same. Freshly installed, you see the GUI exactly one Time.

**Comment by hellwolf** [comment]
Hey folks, it seems a hard nut to crack. I am scratching my head why this is happening. I have looked into buildFHSBubblwrap works even.

If you can get a system-tray plugin for your desktop environment, you may have the luck of being able to open the interface for you.

Until then, any help is welcome. I will keep digging another day!

**Comment by ajmasia** [comment]
From the system tray it opens, but since the icon is not painted you have to guess where the click is. Is there any way to launch the click action from the terminal?

Thanks @hellwolf for investigate it!

**Comment by hellwolf** [comment]
I should have fixed the status not shared issue in #251139

**Comment by hellwolf** [comment]
I won't be able to fix the "No systry icon appears" issue still, it's a known bug and I suspect it's a defect in upstream closed source code.

**Comment by hellwolf** [comment]
Expecting https://github.com/NixOS/nixpkgs/pull/251565 to be available next week ish for both 23.05 and unstable.

**Comment by hellwolf** [comment]
FYI: Also, a new package for nautilus users: https://github.com/NixOS/nixpkgs/pull/302422 please give a :+1: for faster merging!
